metals/src/main/scala/scala/meta/internal/metals/MetalsHttpClient.scala

```diff
@@ -184,7 +184,11 @@ final class MetalsHttpClient(
 
   override def metalsPublishDecorations(
       params: PublishDecorationsParams
-  ): Unit = ()
+  ): Unit = {
```

Comment:
This was causing worksheet decorations not to work for clients that had `isHttpEnabled` set to true.